snapcraft/commands/remote.py

```diff
@@ -131,6 +131,8 @@ def run(self, parsed_args: argparse.Namespace) -> None:
         ):
             raise AcceptPublicUploadError()
 
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
`disable-next` maybe?

Reply:
Which do you prefer?
```python
# pylint: disable-next=attribute-defined-outside-init
self._snapcraft_yaml = yaml_utils.get_snap_project().project_file
# pylint: disable-next=attribute-defined-outside-init
self._parsed_args = parsed_args
```

```python
# pylint: disable=attribute-defined-outside-init
self._snapcraft_yaml = yaml_utils.get_snap_project().project_file
self._parsed_args = parsed_args
# pylint: enable=attribute-defined-outside-init
```

Reply:
Method 2 is probably the most readable? Not a strong opinion though.

Reply:
yeap either-or is fine with me